packages/common-all/src/util/stringUtil.ts

```diff
@@ -1,8 +1,24 @@
 import levenshtein from "fast-levenshtein";
+import { CONSTANTS } from "../constants";
 
 /**
  * Returns levenshtein distance between the two strings, the higher the number
  * the further apart the strings are. 0 signals that the strings are equal. */
 export function levenshteinDistance(s1: string, s2: string): number {
   return levenshtein.get(s1, s2);
 }
+
+export function parseDendronURI(linkString: string) {
```

Comment:
I actually moved that from LinkUtils to here. Since `LinkUtils` is part of `engine-server`, I felt it was easier to just move this function than the entire class